style/table-types.css

```diff
@@ -179,29 +186,35 @@
 .add-new-label {
   display: flex;
   gap: 10px;
+  border-radius: 3px;
   align-items: center;
-  justify-content: space-around;
-  padding-left: 5px;
-  padding-bottom: 5px;
+  padding: 4px 8px 4px 11px;
   font-size: 13px;
   justify-content: flex-start;
   cursor: pointer;
 }
+.add-new-label .label-color {
+  margin-left: 4px;
+}
+
+.add-new-label:hover {
+  background-color: #313740;
+}
 
 .new-label-title,
 .labels-chooser-container .label-title {
-  width: 155px;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
 }
-.labels-chooser-container .label-title {
-  width: 125px;
-}
 
 .labels-search {
   min-width: 200px;
-  border-radius: 4px;
+  border-radius: 12px !important;
```

Comment:
Is it possible to keep tables `!important` free? let's try to move that to the editor